app/src/main/java/com/tari/android/wallet/ffi/FFIBase.kt

```diff
@@ -32,6 +32,8 @@
  */
 package com.tari.android.wallet.ffi
 
+import com.orhanobut.logger.Logger
```

Comment:
Is this import used?

Reply:
No it's not used, removed, thanks. PR update coming with another small improvement.